### ISSUE TITLE
Dependent oedema

### ISSUE THREAD

**Issue by tudorgroza** [obophenotype/human-phenotype-ontology#6085]
**Preferred term label:**
Dependent oedema

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
HP:0000969?

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
I do not think this should be a new term. "Dependent" basically means that the edema is related to gravity -- there is more edema in the feet that on the scalp, for instance. Therefore, this is a related synonym for "pedal edema". If a patient is bed-ridden, then the feet are no longer "dependent" and there might be edema in the back, for instance. But basically there is no "non-dependent" edema, so this term does not add anything.
